Sources/TuistSupport/Extensions/AbsolutePath+Extras.swift

```diff
@@ -60,6 +60,15 @@ extension AbsolutePath {
     public func sha256() -> Data? {
         try? SHA256Digest.file(at: url)
     }
+
+    public func md5() throws -> String {
```

Comment:
I'd move those methods to [FileHandler](https://github.com/tuist/tuist/blob/master/Sources/TuistSupport/Utils/FileHandler.swift) and keep `AbsolutePath` as business-logic-free as possible. That way we can sub the methods easily and change the behavior for testing purposes:

```swift
protocol FileHandling {
  func md5(path: AbsolutePath) throws -> String
  func base64MD5(path: AbsolutePath) throws -> String
}
```

Reply:
Sounds good, I'll move them